server/schedulers/hot_region.go

```diff
@@ -472,6 +472,16 @@ func (bs *balanceSolver) tryAddPendingInfluence() bool {
 	if bs.best == nil || len(bs.ops) == 0 {
 		return false
 	}
+
+    // Because there is no region data on the TiFlash ReadNodes,
+    // so neither srcStore nor dstStore should be tiflash_mpp node.
+    if bs.best.srcStore.IsTiFlashMPP() || bs.best.dstStore.IsTiFlashMPP() {
+        log.Error("Got hot region(srcStore) in tiflash_mpp node or scheduling to tiflash_mpp node",
+            zap.Bool("srcStore is tiflash_mpp", bs.best.srcStore.IsTiFlashMPP()),
+            zap.Bool("dstStore is tiflash_mpp", bs.best.dstStore.IsTiFlashMPP()))
```

Comment:
I think adding metrics here is better, just like line 486. A log is not that helpful and necessary.

Reply:
I thinks it's ok to add metrics. But why logging is not helpful, we can check log to know this error situation?

Reply:
done